### ISSUE TITLE
Implement Regression Smoke Tests

### ISSUE THREAD

**Issue by alexpappasoddball** [department-of-veterans-affairs/va.gov-team#11128]
In order to catch future changes to PDF serializations, we want to write a regression spec which tests a serialized response with a PDF field name.

Implementation will most likely be an Rspec request or session test.

**Comment by alexpappasoddball** [comment]
@f1337 can you update this to reflect the actual work we are doing here.

**Comment by johnpaulashenfelter** [comment]
I think at a minimum, we want a spec for validPdf comparable to the PACT version so we can compare and contrast. I'll do that now.

**Comment by f1337** [comment]
Yep, per @johnpaulashenfelter's comment, at minimum we want a request spec that validates the serialization.

**Comment by alexpappasoddball** [comment]
@f1337 can you add your comments and close this one!

**Comment by f1337** [comment]
Per my comments on this [draft pull-request](https://github.com/department-of-veterans-affairs/vets-api/pull/4554):

A PACT test *could* work for such regressions, but I wouldn't advise it. PACT tests aren't exactly lightweight to implement, and they're not intended for unit-level regressions.

The work we did to prove out the concept has been merged into the PACT forms branch/work.

We can close this story as "done" and "decided".
